README.md

```diff
@@ -44,6 +44,10 @@ Votes on a PR are sourced through the following mechanisms:
 * A :+1: or :-1: reaction on the PR itself
 * An accept/reject [pull request review](https://help.github.com/articles/about-pull-request-reviews/)
 * The PR itself counts as :+1: from the owner
+* Voting goes on for the duration of the voting window - currently 2 or 3 hours,
+  depending on the local server time.
+* While the voting process is going, users can change their reactions and edit
+  their comments, effectively changing their vote.
```

Comment:
You're right. Will edit the heading to reflect the meaning of the bullet list better.